### PR TITLE
docs: added config.json and directory refactored

### PR DESCRIPTION
**Pull request by iamayushdas**
config.json added
refactored directories

### CODE REVIEW

**Comment by iamayushdas** [comment]
@juzhiyuan have a look

File: docs/en/latest/config.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "version": 1.0,
```
Comment (spacewander) [inline docs/en/latest/config.json:2]: It seems @tzssangglass wants to make a `0.1` release:
https://github.com/apache/apisix-java-plugin-runner/pull/35#issuecomment-864721810

File: docs/en/latest/config.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    "sidebar": [
+      {
+        "type": "category",
+        "label": "Installation",
+        "items": [ "development", "how-it-works", "the-internal-of-apisix-java-plugin-runner" ]
```
Comment (spacewander) [inline docs/en/latest/config.json:7]: IMHO, each item should have separate sidebar entry, and they are not all about "Installation".

Comment (iamayushdas) [reply docs/en/latest/config.json:7]: Okay captain

**Review by juzhiyuan — COMMENTED** [review]
LGTM except the version and sidebar :)

**Comment by iamayushdas** [comment]
> LGTM except the version and sidebar :)

fixed, but what's the conflicts here

File: docs/en/latest/config.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+    "version": 0.1,
+    "sidebar": [
+      {
+        "type": "category",
```
Comment (spacewander) [inline docs/en/latest/config.json:5]: The type is incorrect, see: https://github.com/apache/apisix/blob/2707fda3eb7b23d995ac23ecf4a17000dd765855/docs/en/latest/config.json#L164 as an example

**Review by spacewander — CHANGES_REQUESTED** [review]

File: docs/en/latest/config.json (hunk 3 above)
Comment (iamayushdas) [reply docs/en/latest/config.json]: oh yes , I forgot, it done now

File: docs/en/latest/config.json [hunk 4]
```diff
@@ -1,10 +1,20 @@
...
       {
         "type": "category",
-        "label": "Installation",
-        "items": [ "development", "how-it-works", "the-internal-of-apisix-java-plugin-runner" ]
+        "label": "Development",
+        "id": "development"
+      },
+      {
+        "type": "category",
+        "label": "Working",
```
Comment (juzhiyuan) [inline docs/en/latest/config.json]: 🤔 Not sure if we have a better label?

Comment (iamayushdas) [reply docs/en/latest/config.json]: I thought this will be more apt.

**Review by spacewander — APPROVED** [review]

**Comment by tzssangglass** [comment]
hi, pls resolve conflicting files

**Comment by iamayushdas** [comment]
> hi, pls resolve conflicting files
> 
> 

I don't think conflicts are from my side

**Comment by tzssangglass** [comment]
> > hi, pls resolve conflicting files
> 
> I don't think conflicts are from my side

try pulling the latest code from the `main` branch of upstream?
I see this 
image

**Comment by juzhiyuan** [comment]
@iamayushdas Because this PR is out of date from main branch :) so we have to resolve those conflicts.

**Comment by iamayushdas** [comment]
resolving conflict in new PR, CLOSING THIS

**Pull request closed without merging**